### ISSUE TITLE
ColorPicker: Form elements must have labels

### ISSUE THREAD

**Issue by WTW-Steve-Martin** [primefaces/primeng#14941]
### Describe the bug

The Axe Dev Tools for Accessibility is reporting the following issue:
![image](https://github.com/primefaces/primeng/assets/153538315/e8c8a157-3226-4cc9-9d37-28f2cd47db60)

Axe is highlighting that the color picker doesn't have an Aria Label.

Please could we update this control so that the Axe scan passes?

### Environment

Windows 10

### Angular version

17.2.3

### PrimeNG version

17.9.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

### Steps to reproduce the behavior

1. Install the Axe Dev Tools for Accessibility plug-in.
2. Visit the PrimeNG dropdown page (https://primeng.org/colorpicker) on Chrome/Edge browser.
3. Scan the page with Axe

### Expected behavior

The Axe scan doesn't return this error.
